src/Dfc.CourseDirectory.Web/Controllers/QualificationsController.cs

```diff
@@ -1,149 +1,34 @@
 ﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Threading.Tasks;
-using Dfc.CourseDirectory.Common;
-using Dfc.CourseDirectory.Services.CourseService;
-using Dfc.CourseDirectory.Services.Interfaces.CourseService;
-using Dfc.CourseDirectory.Services.Interfaces.VenueService;
 using Dfc.CourseDirectory.Web.ViewModels;
 using Microsoft.AspNetCore.Authorization;
 using Microsoft.AspNetCore.Http;
 using Microsoft.AspNetCore.Mvc;
-using Microsoft.Extensions.Logging;
 
 namespace Dfc.CourseDirectory.Web.Controllers
 {
     [Authorize("Fe")]
     public class QualificationsController : Controller
     {
-        private readonly ILogger<QualificationsController> _logger;
         private readonly IHttpContextAccessor _contextAccessor;
-        private ISession _session => _contextAccessor.HttpContext.Session;
-        private readonly ICourseService _courseService;
-        private readonly IVenueService _venueService;
 
+        private ISession _session => _contextAccessor.HttpContext.Session;
 
-        public QualificationsController(
-            ILogger<QualificationsController> logger,
-            IHttpContextAccessor contextAccessor, ICourseService courseService, IVenueService venueService)
+        public QualificationsController(IHttpContextAccessor contextAccessor)
```

Comment:
Don't need `IHttpContextAccessor`

Reply:
It's being used by the `ISession` accessor.

Reply:
Updated to use `HttpContext.Session`